test/include/dlaf_test/matrix/util_generic_blas.h

```diff
@@ -50,25 +50,25 @@ using namespace dlaf_test;
 ///
 template <class ElementIndex, class T>
 auto getLeftTriangularSystem(blas::Uplo uplo, blas::Op op, blas::Diag diag, T alpha, SizeType m) {
-  bool op_a_lower = false;
-  if ((uplo == blas::Uplo::Lower && op == blas::Op::NoTrans) ||
-      (uplo == blas::Uplo::Upper && op != blas::Op::NoTrans))
-    op_a_lower = true;
+  const bool op_a_lower = ((uplo == blas::Uplo::Lower && op == blas::Op::NoTrans) ||
+                           (uplo == blas::Uplo::Upper && op != blas::Op::NoTrans))
+                              ? true
+                              : false;
```

Comment:
No need to use ternary operator here. Just assign the boolean value.
```suggestion
  const bool op_a_lower = (uplo == blas::Uplo::Lower && op == blas::Op::NoTrans) ||
                          (uplo == blas::Uplo::Upper && op != blas::Op::NoTrans);
```